spec/win32/unit/certstore_spec.rb

```diff
@@ -98,6 +98,95 @@
     end
   end
 
+  describe "#cert_get" do
+    context "When passing empty certificate store name" do
+      let (:store_name) { "" }
+      it "raises ArgumentError" do
+        expect { certstore.open(store_name) }.to raise_error("Invalid Certificate Store.")
```

Comment:
we should check the exception type here, i.e. `.to raise_error(ArgumentError, "Invalid Certificate Store")`